Makefile

```diff
@@ -272,25 +269,21 @@ vbox: vbox-create
 
 .PHONY: docs
 docs:
-	@echo Generating docs according to the Doxyfile...
+	@printf "$(COLOR_INFO)Generating docs according to the Doxyfile...$(COLOR_NONE)\n"
 	@doxygen ./Doxyfile
 
-.PHONY: todo
-todo:
-	-@for file in $(ALLFILES:Makefile=); do fgrep -i -H --color=always -e TODO -e FIXME $$file; done; true
-
```

Comment:
Why remove this?

Reply:
It didn't work anymore, and I don't think anyone ever used it.